CONTRIBUTING.md

```diff
@@ -100,6 +124,9 @@ bazel test -c opt -k \
 All submissions, including submissions by project members, require review. We
 use Github pull requests for this purpose.
 
+For Visual Diffs we prefer to using [reviewNB](www.reviewnb.com). It Simplifies your Notebook Workflows.
```

Comment:
We can probably just remove this line. It's true but it is generally useful for reviewers and not specific to the Dev Environment.